doc/data/messages/b/bad-except-order/good.py

```diff
@@ -0,0 +1,6 @@
+try:
+    print(int(input()))
+except TypeError:
+    raise
+except Exception:
+    raise
```

Comment:
This would emit `try-except-raise`, I think, so maybe we should model doing something with the exception?

Reply:
Based on this discussion https://github.com/PyCQA/pylint/pull/5992#issuecomment-1079794958 we should prefer simplicity/readability over code cleanness.

Reply:
That's fair, but I think a flexible approach to this is worth considering, because I suspect we'll get an issue someday "I rewrote my code the way pylint said to but then I got another error".

Reply:
I leave the decision to @Pierre-Sassoulas  and @DanielNoord

Reply:
How about a ``pass`` instead of a raise ? Still simple, but no ``try-except-raise`` ?

Reply:
I guess it's a losing battle because bare-exception is also in play (probably?). So I'm willing to leave this be 😄